guardian/typerighter
# Add pre-commit hook for Linting

## What does this change?

This PR adds a pre-commit hook which will auto format staged changes using ScalaFMT. It also adds a setup script whose role is sym-linking the script to the hooks dir.




## How to test

Check it out, run the script, make a change and commit. The linter should run against your Scala files. 



## How can we measure success?



## Have we considered potential risks?

Yep



## Images
